src/main/java/strategy/ReturnOperation.java

```diff
@@ -0,0 +1,11 @@
+package strategy;
+
+import core.basesyntax.FruitTransaction;
+import core.basesyntax.Storage;
+
+public class ReturnOperation implements OperationHandler {
+    @Override
+    public void handle(FruitTransaction transaction, Storage storage) {
+        storage.increaseQuantity(transaction.getFruit(), transaction.getQuantity());
```

Comment:
Ensure that the `increaseQuantity` method in `Storage` is correctly handling the increase in quantity, as we do not want to allow adding a negative quantity which could happen during a return operation. You may want to add a check inside the `increaseQuantity` method to verify that the quantity being added is positive.